### ISSUE TITLE
Add schema for Kustomization

### ISSUE THREAD

**Issue by MPV** [SAP/ytt-lint#2]
Would you be open to adding schema for `Kustomization`?

As `kind: Kustomization` is a part of kubernetes (at least via kubectl).

I currently get this warning on these kinds of manifests:

>Error loading schema for kind Kustomization: could not find schema file k8s/kustomize.config.k8s.io/v1beta1/kustomization.json in: [/Users/myself/.ytt-lint/schema /Users/myself/.vscode/extensions/phil9909.ytt-lint-0.2.0/schema]

...on a manifest with:

```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
# [...]
```

**Comment by phil9909** [comment]
Sorry for the delay, when I moved this repo from my user to the SAP org I forgot to activate the "watch" option of GitHub.

Yes, if it is official kubernets, I think it should be part of ytt-lint. Will have a look soon.

**Comment by phil9909** [comment]
With the latest release [v0.2.1](https://github.com/SAP/ytt-lint/releases/tag/v0.2.1) I incorporated the kustomization.yaml schema from [JSON Schema Store](https://www.schemastore.org/json/).
Let me know if this works for you.

**Comment by MPV** [comment]
Wonderful, huge thanks! I can confirm that it worked like a charm. 👍
